tools/generator-bot-template-generator/yarn.lock

```diff
@@ -0,0 +1,8528 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we even need this? This is all pure Node, no?

Reply:
https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
Not sure I understand, we need a yarn.lock file (or package.lock) for any solution that has dependencies right? There arent many dependencies but there are some. 

I did notice I had both yarn.lock and package-lock which is a mistake so I have removed yarn.lock in favor of npm's package.lock and also removed unused dependencies. Good catch, assuming this is what you meant :)